framework_tvb/tvb/interfaces/web/controllers/project/project_controller.py

```diff
@@ -515,11 +516,14 @@ def editstructure(self, project_id=None, first_level=None, second_level=None,
         self._mark_selected(selected_project)
         data = self.project_service.get_filterable_meta()
         filters = StaticFiltersFactory.build_datatype_filters(selected=visibility_filter)
+        tumor_import_algorithm = dao.get_algorithm_by_module('tvb.adapters.uploaders.tumor_dataset_importer',
+                                                             'TumorDatasetImporter')
```

Comment:
You can use the TumorDatasetImporter.__module__ and TumorDatasetImporter.__class__ instead of hardcoded strings.